src/Etg.Yams.Core/YamsConfigBuilder.cs

```diff
@@ -5,14 +5,14 @@ namespace Etg.Yams
 {
     public class YamsConfigBuilder
     {
-        public YamsConfigBuilder(string superClusterId, string clusterId, string instanceUpdateDomain, 
+        public YamsConfigBuilder(string clusterId, string instanceUpdateDomain, 
             string instanceId, string applicationInstallDirectory)
         {
-            this._superClusterId = superClusterId;
             _clusterId = clusterId;
             _instanceUpdateDomain = instanceUpdateDomain;
             _instanceId = instanceId;
             _applicationInstallDirectory = applicationInstallDirectory;
+            _superClusterId = null;
```

Comment:
why not set this to clusterId here?

Reply:
Because if someone calls `SetSuperClusterId(null)` we will end up with `null` in the config. I assume you want to avoid it and always default to Cluster Id

Reply:
`SetSuperClusterId(null)` can throw a NullArgumentException. I think setting it to clusterId by default is cleaner. Caller can then choose to override if they which to do so. It's also consistent with how the other settings are being set to default in the constructor.